test/e2e/specs/realrp/vnetpeering.go

```diff
@@ -0,0 +1,125 @@
+package realrp
+
+import (
+	"context"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-06-01/network"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+
+	"github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
+	"github.com/openshift/openshift-azure/pkg/fakerp/client"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	tlog "github.com/openshift/openshift-azure/test/util/log"
+)
+
+var _ = Describe("Peer Vnet tests [Vnet][Real][LongRunning]", func() {
```

Comment:
Probably unexpected but the [Real] tag is causing this test to run as part of `make e2e-prod`. I think we need to keep them separate.

Reply:
I was following instructions from other reviewers.  We can add them back